### PR TITLE
Add HP Elitebook 2760p support

### PR DESCRIPTION
**Pull request by AlexCatze**

### CODE REVIEW

File: hwdb.d/60-keyboard.hwdb [hunk 1]
```diff
@@ -602,6 +602,18 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHP*G60*Notebook*PC:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pn*2570p*:*
  KEYBOARD_KEY_f8=wlan                                   # Wireless HW switch button
 
+#Elitebook 2760p
+evdev:input:b0011v0001p0001eAB41*
```
Comment (poettering) [inline hwdb.d/60-keyboard.hwdb]: hmm why a match not based on DMI?

Comment (AlexCatze) [reply hwdb.d/60-keyboard.hwdb]: I tried it, but it not worked for me.

Comment (poettering) [reply hwdb.d/60-keyboard.hwdb]: Well, you are matching vendor id 1 and product id 1, which looks very suspicious.

Comment (AlexCatze) [reply hwdb.d/60-keyboard.hwdb]: Okay, I found working dmi match.

File: hwdb.d/60-keyboard.hwdb [hunk 2]
```diff
@@ -602,6 +602,18 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHP*G60*Notebook*PC:*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pn*2570p*:*
  KEYBOARD_KEY_f8=wlan                                   # Wireless HW switch button
 
+#Elitebook 2760p
```
Comment (bluca) [inline hwdb.d/60-keyboard.hwdb]: missing space after `#`

**Pull request merged**